nixos/modules/services/x11/desktop-managers/gnome3.nix

```diff
@@ -151,6 +151,7 @@ in {
     services.colord.enable = mkDefault true;
     services.packagekit.enable = mkDefault true;
     hardware.bluetooth.enable = mkDefault true;
+    services.hardware.bolt.enable = mkDefault true;
```

Comment:
Are we sure we want to enable this by default?

Reply:
I think so; otherwise the Thunderbolt Settings panel in GNOME is completely broken and shows a misleading error message.

Bolt is enabled by default in Fedora. It is not enabled by default in Arch or SuSE, which appears to have confused users, based on Reddit and forum threads I found when trying to fix this issue myself.

And my understanding is that the daemon is only actually started [when udev sees a thunderbolt device is connected](https://gitlab.freedesktop.org/bolt/bolt/blob/0.7/data/90-bolt.rules#L13-14), so the impact should be minimal on other systems.

Reply:
Sounds sane, :+1:  on not confusing users with unexpected things.

Reply:
More concretely, my Lenovo laptop shipped with Thunderbolt 3 security enabled by default, rendering the ports inoperable without a daemon like `bolt` or manual fiddling with the `tbtadm` tool.

GNOME users would expect the Settings dialog to work to enable these devices.

Reply:
Yeah, tested in a VM and the daemon dosen't start on startup. It does start up when going to the thunderbolt panel, but it's a very light process. So :+1: on erring on the side of out of the box experience.

Reply:
What is the size impact?

Reply:
Bolt itself is only `445.4K`, and the other dependencies (`glib` `systemd` and `polkit`) should already be needed. From `nix path-info -f . bolt --human-readable  --size --recursive` it doesn't look like the test dependencies are drawn in by accident.